api/src/main/kotlin/com/gmtkgamejam/ApplicationCallExtensions.kt

```diff
@@ -0,0 +1,24 @@
+package com.gmtkgamejam
+
+import com.auth0.jwt.JWT
+import com.gmtkgamejam.models.auth.AuthTokenSet
+import com.gmtkgamejam.services.AuthService
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+
+fun ApplicationCall.getAuthTokenSet(authService: AuthService): AuthTokenSet? {
+    return this.request.header("Authorization")
+        ?.substring(7)
```

Comment:
While this is probably clear for most that it's removing the "Bearer " part of the authorization header in Kotlin there are options to make this intention clearer.

```suggestion
        ?.removePrefix("Bearer ")
```

or 

```suggestion
        ?.substringAfter("Bearer ")
```

Reply:
Updated 👍🏻